lib/elixir/lib/task.ex

```diff
@@ -526,15 +532,15 @@ defmodule Task do
   The shutdown method is either a timeout or `:brutal_kill`. In case
   of a `timeout`, a `:shutdown` exit signal is sent to the task process
   and if it does not exit within the timeout it is killed. With `:brutal_kill`
-  the task is killed straight away. In case the task exits abnormally, or a 
+  the task is killed straight away. In case the task exits abnormally, or a
   timeout shutdown kills the task, this function will exit with the same reason.
```

Comment:
It is ok to remove that bit. It means if the task is shutdown or killed by another process.